src/elements/content-sharing/SharingNotification.js

```diff
@@ -259,7 +259,7 @@ function SharingNotification({
             closeComponent();
         },
         setIsLoading,
-        transformRequest: data => convertCollabsRequest(data),
+        transformRequest: data => convertCollabsRequest(data, collaboratorsList),
```

Comment:
It seems like this filtering logic should live in the `handleSuccess` call above or even during render rather than a util since it's accessing component state.

Reply:
I just saw it's accessing a prop, not state. I still think it'd best be served for the logic to live in the component rather than a util, but not going to hold up the merge for it.